examples/experimental/CrypTen/CrypTen - Traing Encrypted Neural Network - Plans.ipynb

```diff
@@ -0,0 +1,543 @@
+{
```

Comment:
<p>typo "... where each worker has a subset of the features we need for the training..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/3822/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>